src/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead>
+        <tr>
+          {Object.keys(props.results[0]).map(function(key) {
+            return (
+              <th key={(-Math.random() * 10000000000).toString()} scope="col">
+                {key}
+              </th>
+            );
+          })}
+          <th scope="col">Number of nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map(result => {
+          return (
+            <tr key={(Math.random() / 10000000000).toString()}>
+              {Object.keys(result).map(function(key) {
+                return (
+                  <td key={(Math.random() * 10000000000).toString()}>
+                    {result[key]}
+                  </td>
```

Comment:
I am not sure what you are trying to do by using a random number generator to give the rows and data cells keys, however you don't need to do anything complicated as the bookings each have a unique id field. You can use this value for your key. You also shouldn't need to give each cell a unique identifier, as it is already grouped with the rest of the data for that booking.